### ISSUE TITLE
Add HDT+ 20 pin connector specs

### ISSUE THREAD

**Issue by MatthewTingum** [Necrosys/x86-JTAG-Information#4]
Images of the Supermicro M12SWA-TF motherboard found on google show a connector identical to the one shown [here](https://github.com/Necrosys/x86-JTAG-Information/blob/3cfd7d9cf8cedd9155091c2f838aa27378849576/Connector/HDT/HDTPlus_P.jpg). Document the connector specifications and compatible receptacles / cables.

Document the pinout. I would assume it should be possible to determine the JTAG pins at a minimum.

**Comment by MatthewTingum** [comment]
A list of devices with this header exposed would be great too! It's not clear where the image in OP came from.

**Comment by Necrosys** [comment]
> Document the connector specifications and compatible receptacles / cables.
> Document the pinout.

Partially done in below commit.
Commit: https://github.com/Necrosys/x86-JTAG-Information/commit/7fa683b5dc0e24f9716461bb1f15da32f76d5630

> It's not clear where the image in OP came from.
Done in below commit.
Commit: https://github.com/Necrosys/x86-JTAG-Information/commit/702a1d87d8114f9255f12b234da20b87ac85adaf

**Comment by MatthewTingum** [comment]
I'd consider this issue closed but I'll leave it up to you to hit the button. You've provided some great details!. I have a few things to comment but you might have to take them by word of mouth because I can't seem to find my sources.

I've got some motherboards with (what I believe to be) HDT+ headers and the 2x10 1.27mm connectors I tested fit perfectly. I'll be testing base JTAG on older processors when the breakouts I designed are fabricated. That should be a redundant validation of the pinout.

I can't find the link anymore but I found a datasheet that specifies the HDT+ header as being a Samtec `AS-137098-05`. Ordered a handful from aliexpress to populate some old boards. Will report back when I populate an `LA-8868P` motherboard.
